### PR TITLE
State handler

### PR DESCRIPTION
**Pull request by swehner**
Allow all plugins to implement save() and load() methods to save state when shutting down and loading it once foos is started again.

### CODE REVIEW

**Comment by dpaneda** [comment]
Good idea! I'll check the PR today.

File: foos/plugin_handler.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+logger = logging.getLogger(__name__)
+print(logger.level)
+print(logger.parent, logger.parent.level)
+
+
+class PluginHandler:
+    def __init__(self, bus):
+        self.status_file = '.status'
+        print("PH1", logger, logger.level, logger.handlers)
+        print("PH1", logger.parent, logger.parent.level, logger.parent.handlers)
```
Comment (dpaneda) [inline foos/plugin_handler.py]: Are those some forgotten debug logs or you leave them on purpose?

Comment (swehner) [inline foos/plugin_handler.py]: Totally forgot about them - I was having trouble with the logging module... I'll delete them

**Comment by dpaneda** [comment]
All seems good to me

**Comment by swehner** [comment]
You're right - I don't it's worth it...

**Pull request closed without merging**